src/compiler/VendorCompiler.ts

```diff
@@ -147,6 +168,9 @@ export class VendorCompiler implements ICompiler {
         return {
             mode: 'production',
             entry: path.resolve(this.assetsPath, CplaceTypescriptCompiler.DEST_DIR, VendorCompiler.VENDOR_ENTRY),
+            externals: {
```

Comment:
@vladimir-arsov Shouldn't this be in the `vendor.config.js`?

Reply:
Maybe not. It might be needed in other plugins that have 3rd party libraries requiring jquery.